tests/Unit/Framework/ActionTesting.hpp

```diff
@@ -818,7 +820,7 @@ class MockDistributedObject {
       NAME##_queue_.push_back(                                                \
           std::make_unique<BOOST_PP_IF(USE_SIMPLE_ACTION, InvokeSimpleAction, \
                                        InvokeThreadedAction) < Action,        \
-                           Args...>> (this, std::move(args)));                \
+                           Args...> > (this, std::move(args)));               \
```

Comment:
why these space changes?

Reply:
I think this is clang-tidy gone rogue. I'll revert.